### ISSUE TITLE
Styling changes when uploading files

### ISSUE THREAD

**Issue by erickpeirson** [arXiv/arxiv-submission-ui#41]
When I upload a new source package, I see this:
![image](https://user-images.githubusercontent.com/3451594/48771052-3bf19780-ec8e-11e8-89ec-0b397e72041f.png)

- They should all be green
- I don't see any gray stripes

But then when I reload the page, I see:

![image](https://user-images.githubusercontent.com/3451594/48771092-54fa4880-ec8e-11e8-9e47-df9bf13ecf58.png)

That looks a lot better. But then I upload another file...

![image](https://user-images.githubusercontent.com/3451594/48771163-92f76c80-ec8e-11e8-8326-5255df85c93e.png)

- No striping, no green.

**Comment by erickpeirson** [comment]
Ping @eawoods : I'm going to see if I can figure this out quickly, but wanted to make sure there was a record in case I screw things up.... ;-)

**Comment by erickpeirson** [comment]
Clue number 1: the non-green rows have class ``is-successeven``

**Comment by eawoods** [comment]
I've got your back if you need another set of eyes. :)
It looks like the logic on style striping is messed up, and that might be my fault

The green is supposed to be superseding gray if files are new, but not touch it otherwise

**Comment by eawoods** [comment]
clue #1 may be the entirety of it

**Comment by erickpeirson** [comment]
Here's what it looks like when I add a space:
![image](https://user-images.githubusercontent.com/3451594/48771392-4c564200-ec8f-11e8-934a-04de642fb470.png)

**Comment by eawoods** [comment]
huh. Better but different

**Comment by erickpeirson** [comment]
I mean, that looks kind of right to me: it's weird if the stripes are there sometimes and other times not. And the green looks right.

**Comment by eawoods** [comment]
Yeah, I've been mulling over that (for the past thirty seconds, ha) and it does look right. Let's go with it!

**Comment by erickpeirson** [comment]
But the padding is different; is that intentional?
![image](https://user-images.githubusercontent.com/3451594/48771493-8de6ed00-ec8f-11e8-9bc2-51e883ed25b0.png)

**Comment by erickpeirson** [comment]
![image](https://user-images.githubusercontent.com/3451594/48771514-9ccd9f80-ec8f-11e8-8f5c-efe8f22ff832.png)

**Comment by eawoods** [comment]
nope the padding difference looks like an unintended consequence

**Comment by eawoods** [comment]
I wonder if it's a byproduct of me trying to get the subdirectory padding to look good. There's an inheritance rule -- did anything change about the way li items are nested in a one-layer list?

**Comment by erickpeirson** [comment]
I don't see anything like that. Could it be that it's just the difference between having ``is-success`` and not?

```
li.is-success {
    background-color: #f8fdf6;
    color: #1F6B0E;
    border: 1px solid #3CB521;
    padding: 2px;
}
```

**Comment by erickpeirson** [comment]
If I disable the padding from ``.is-success``:

![image](https://user-images.githubusercontent.com/3451594/48771643-f46c0b00-ec8f-11e8-8d63-4f82b1b9646f.png)

**Comment by eawoods** [comment]
not that, then - it's on the gray ones
I said padding but it's actually margin

**Comment by erickpeirson** [comment]
So here's what it looks like if I add ``padding: 2px`` to ``ol.file-tree li``

![image](https://user-images.githubusercontent.com/3451594/48771803-4e6cd080-ec90-11e8-888d-8040119534aa.png)

**Comment by eawoods** [comment]
It may be helpful to look at the inspector and see where the white space is coming from -- and make sure we know what we want to do with it. Applying a rule that's too global is going to have possible effects on nested subdirectories that we don't want.

**Comment by erickpeirson** [comment]
Ok, I pushed the fix for the smooshed class to develop. I'd better hand this off to an expert... ;-)

https://github.com/cul-it/arxiv-submission-ui/commit/312ae34c3531222f0339b304ecaedded5f3f6dc9

**Comment by eawoods** [comment]
* grin * shall I write myself an issue or just take a look at it as the first thing of the day?

**Comment by erickpeirson** [comment]
Your call, totally not an emergency, so however it fits

**Comment by erickpeirson** [comment]
This is fixed as of 95693aa
